src/ng/parse.js

```diff
@@ -1703,11 +1703,11 @@ function $ParseProvider() {
 
   this.$get = ['$filter', '$sniffer', function($filter, $sniffer) {
     var $parseOptions = {
-          csp: $sniffer.csp,
+          csp: $sniffer.csp && $sniffer.csp.noUnsafeEval,
```

Comment:
It would be except that there are loads of tests that mock out sniffer and don't provide a `csp` object. I debated back and forth between fixing the tests or fixing this line and in the end I felt that this was simpler as it meant all these other tests didn't need to know about the csp object.

You can see this in action if you remove this check and then run `grunt autotest`